sidebar.php

```diff
@@ -26,17 +26,17 @@
 		<iframe src="//widgets.spinitron.com/widget/now-playing-v2?station=kbcs&num=5&sharing=1&cover=1&player=1&merch=0&non-music=1" width="100%" frameborder="0" allow="encrypted-media"></iframe>
 	</div>
 
-	<div id="social-links">
-		<div>
+	<nav id="social-links">
```

Comment:
See earlier comments regarding labelling navs

Reply:
Fixed in commit.